src/sound/backend/oalsound.cpp

```diff
@@ -2170,7 +2170,7 @@ void OpenALSoundRenderer::UpdateListener(SoundListener *listener)
 			{
 				ALuint source = GET_PTRID(schan->SysChannel);
 				if (source && !(schan->ChanFlags & CHAN_UI))
-					alSourcef(source, AL_PITCH, schan->Pitch / 128.0f);
+					alSourcef(source, AL_PITCH, PITCH_MULT);
```

Comment:
`schan->Pitch / 128.0f * PITCH_MULT`

Reply:
Okay, do I need to do this for the other one? I'd like to make as few adjustment commits as possible.